src/infrastructure/repositories/ViemChainMetricsRepository.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  createPublicClient,
+  http,
+  PublicClient
+} from "viem"
+// @ts-ignore
```

Comment:
what error are you getting here? Ideally, when using `// @ts-ignore` or `// @ts-expect-error`, a short message next to it should be added to understand your future you (And other devs) why the error needed to be ignored/suppressed.

Same applies in line 22